### PR TITLE
Next.js app configuration

### PR DESCRIPTION
**Pull request by iker-barriocanal**
Be more explicit about what to do with the configuration of the app.

### CODE REVIEW

File: src/platforms/javascript/guides/nextjs/manual-setup.mdx [hunk 1]
```diff
@@ -88,7 +88,10 @@ Also keep in mind, by default we disable source map upload in `dev` mode when ru
 
 ## Configure sentry-cli
 
-Under the hood, the `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/). (If you choose to combine the two approaches, the environment variables will take precedence over values set in the properties file.)
+Under the hood, the `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/).
+
+If you choose to combine the two approaches, the environment variables will take precedence over values set in the properties file.
+You can add the properties file to your VCS excluding any sensitive data (like tokens), and add it to the environment variables.
```
Comment (imatwawana) [inline src/platforms/javascript/guides/nextjs/manual-setup.mdx:94]:
```suggestion
You can add the properties file to your VCS, excluding any sensitive data (like tokens), and add it to the environment variables.
```

File: src/platforms/javascript/guides/nextjs/manual-setup.mdx [hunk 2]
```diff
@@ -88,7 +88,10 @@ Also keep in mind, by default we disable source map upload in `dev` mode when ru
 
 ## Configure sentry-cli
 
-Under the hood, the `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/). (If you choose to combine the two approaches, the environment variables will take precedence over values set in the properties file.)
+Under the hood, the `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/).
```
Comment (imatwawana) [inline src/platforms/javascript/guides/nextjs/manual-setup.mdx:91]: "Under the hood" is such an English-specific idiom that I think we should avoid it (I know this wasn't part of your edits, but I figure let's fix this now). Wondering if this text loses anything if we just take it out? 

```suggestion
The `SentryWebpackPlugin` uses `sentry-cli`, which can be configured in one of two ways - using a properties file, or with environment variables - both of which are discussed below. For full details, see [the CLI configuration docs](https://docs.sentry.io/product/cli/configuration/).
```

**Review by imatwawana — APPROVED** [review]
Minor edits and one wording question at line 91. Otherwise, LGTM!

File: src/platforms/javascript/guides/nextjs/manual-setup.mdx (hunk 2 above)
Comment (iker-barriocanal) [reply src/platforms/javascript/guides/nextjs/manual-setup.mdx]: Since I don't think so, applying your suggestion.

**Pull request merged**